CriticalMass/AppController.swift

```diff
@@ -53,15 +53,13 @@ class AppController {
         TwitterManager(networkLayer: networkOperator, request: TwitterRequest())
     }()
 
-    private lazy var mapOverlayErrorHandler: ErrorHandler = {
```

Comment:
I removed that after the rebase since it was not used and I added a `MapInfoViewController` @lennet

Reply:
I also did not get why this has to be injected and the presentInfo function was public?

Reply:
That was the problem I was describing in my PR. There is a circular dependency and by removing it. the code compiles but the error won't be presented

Reply:
Ah :) It feels a little weird. Should we remove it again for now and tackle that later?